src/components/Badge/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactNode } from 'react';
+import PropTypes from 'prop-types';
+import { BaseProps } from '../types';
+import { StyledBadge, StyledBadgeContainer } from './styled';
+
+type badgeVariant = 'default' | 'lightest' | 'outline-brand' | 'brand';
+
+interface Props extends BaseProps {
+    label?: ReactNode;
+    variant?: badgeVariant;
+    style?: object;
```

Comment:
Remove only this line since style is already defined in BaseProps and this interface extends from BaseProps.